aws-crt/aws-crt.csproj

```diff
@@ -52,8 +52,7 @@
 
     <CMakeBinaries>$(ProjectDir)../build</CMakeBinaries>
     <CMakeConfig Condition="$(Configuration) == 'Debug'">Debug</CMakeConfig>
-    <CMakeConfig Condition="$(CMakeConfig) == ''">RelWithDebInfo</CMakeConfig>
-    <CMakeLibCrypto Condition="$(LibCryptoPath) != ''">-DLibCrypto_INCLUDE_DIR=$(LibCryptoPath)/include -DLibCrypto_STATIC_LIBRARY=$(LibCryptoPath)/lib/libcrypto.a</CMakeLibCrypto>
+    <CMakeConfig Condition="$(CMakeConfig) == ''">Release</CMakeConfig>
```

Comment:
Should this go back to RelWithDebInfo now that we're doing a targeted strip?

Reply:
I will do that.

Reply:
This turned out to be necessary for now, until I can perform further upstream size optimizations